me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeGetAccessTokenListener.java

```diff
@@ -0,0 +1,10 @@
+package me.id.webverifylib;
+
+/**
+ * Created by mirland on 13/12/16.
+ * Listener to get the access token
+ */
+public interface IDmeGetAccessTokenListener {
```

Comment:
I don't know, they used this format to the `IDmeWebVerify` class. I think I should follow this format